infrastructure/validators/onboarding/run-validator/using-systemd.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Use Systemd
+description: Using a service manager for your validator node.
+---
+
+You can run your validator as a [systemd](https://en.wikipedia.org/wiki/Systemd){target=\_blank} process so that it will automatically restart on server reboots or crashes (and helps to avoid getting [slashed]()).
+
+Before following this guide you should have already set up your validator by following the [How to Validate]() article.
```

Comment:
I'm using the flag `TODO:update-path` so in the future we can do `Control+f` and find all the pending links that we might have :)

```suggestion
Before following this guide you should have already set up your validator by following the [How to Validate](TODO:update-path) article.
```